examples/simple.rs

```diff
@@ -0,0 +1,93 @@
+//!
+//!  simple.rs
+//!
+//!  John Berry <ulfmagnetics@gmail.com>
+//!
+//!  Always remember to run high performance Rust code with the --release flag. `Synth`
+//!
+
+extern crate dsp;
+extern crate pitch_calc as pitch;
+extern crate portaudio;
+extern crate synth;
+
+use dsp::{Node, Settings};
+use pitch::{Letter, LetterOctave};
+use portaudio as pa;
+
+const CHANNELS: i32 = 2;
+const FRAMES: u32 = 64;
+const SAMPLE_HZ: f64 = 44_100.0;
+
+fn main() {
+  run().unwrap()
+}
+
+fn run() -> Result<(), pa::Error> {
+    // Construct the simplest possible synth --
+    // a single voice containing a sine wave oscillating at 440 Hz.
+    let mut synth = {
+        use synth::dynamic::{Oscillator, Waveform, Amplitude, Frequency, FreqWarp};
+
+        // Create our oscillator at A440
+        let oscillator = Oscillator::new(Waveform::Sine,
```

Comment:
Ah yes, now I see, I missed the `impl Amplitude for f32` and `impl Frequency for f64` blocks before. I hadn't considered that numeric types could implement traits -- very useful in this case!